### PR TITLE
Update toonnongaeoy.dino.icu to my homelab.

### PR DESCRIPTION
**Pull request by toonnongaeoy**
# [Updating] `toonnongaeoy.dino.icu`

## Description

Change to point for my homelab to use reverse proxy to the site, since neocities won't disable ssl from this Slack Message :
https://hackclub.slack.com/archives/C06KMENGHQC/p1716588335739969

### CODE REVIEW

File: dino.icu.yaml [hunk 1]
```diff
@@ -359,8 +359,8 @@ tejas: # by tej
 
 toonnongaeoy:
  - ttl: 600
-   type: A
-   value: 198.51.233.1
+   type: CNAME
+   value: retro.toonstorytime.me
```
Comment (alx-alexpark) [inline dino.icu.yaml:363]: CNAME records need a trailing dot (`.`)

**Review by alx-alexpark — CHANGES_REQUESTED** [review]

File: dino.icu.yaml (hunk 1 above)
Comment (alx-alexpark) [reply dino.icu.yaml:363]: It should be `retro.toonstorytime.me.`

Comment (toonnongaeoy) [reply dino.icu.yaml:363]: Okay.

Comment (toonnongaeoy) [reply dino.icu.yaml]: Fixed. @alx-alexpark

**Review by alx-alexpark — APPROVED** [review]
LGTM!

**Pull request merged**